src/main/java/com/google/edith/servlets/UserInsights.java

```diff
@@ -33,61 +35,64 @@
 
   private String userId;
 
-  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();  
-  private static final DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final DatastoreService datastore = 
+      DatastoreServiceFactory.getDatastoreService();  
```

Comment:
Any reason to not have this var as a class var that gets initialized via the constructor? This will make testing with a mock/fake datastore service easier.

Reply:
No, I made it a class variable.